### DISCUSSION TITLE
EV State in the GUI?

### DISCUSSION THREAD

**Discussion by mif11k** [dingo35/SmartEVSE-3.5#148] [Q&A]
Hi, during recent fw upgrade I suddenly saw EV STATE information in the GUI (web server). Is it actually possible to read this data from the connected car, or is it work in progres? The battery SOC would be really cool!
<img width="1063" alt="Screenshot 2024-10-31 at 17 30 18" src="https://github.com/user-attachments/assets/99706716-6945-4ef7-bb02-5b3041641bac">

**Comment by dingo35** [comment]
That only works if you have a modem connected. You are only supposed to see this if you compile with -DMODEM=1

**Comment by PhilippeRigaux** [comment]
Interesting, where can we find such a modem? By the way I saw it on 3.8.0, after a page refresh it disappeared..

**Comment by dingo35** [comment]
It is a work in progress, the modem will be built in in the new hardware version 4 of the SmartEVSE.
